trade-finance-manager-ui/templates/case/tasks/task.njk

```diff
@@ -13,7 +13,7 @@
     <div class="govuk-grid-row" data-cy="deal-tasks">
 
       <h4 class="govuk-heading-s govuk-!-margin-bottom-2" data-cy="deal-heading">Deal</h4>
-      <p class="govuk-body" data-cy="deal-subheading"><a href="/case/{{ dealId }}/deal" data-cy="deal-subheading-link">{{ deal.details.ukefDealId }}</a> {{ deal.submissionDetails.supplierName }} > {{ deal.submissionDetails.buyerName }}</p>
+      <p class="govuk-body" data-cy="deal-subheading"><a href="/case/{{ dealId }}/deal" data-cy="deal-subheading-link"> <span class="govuk-visually-hidden">View deal </span>{{ deal.details.ukefDealId }} <span class="govuk-visually-hidden">details</span></a> {{ deal.submissionDetails.supplierName }} > {{ deal.submissionDetails.buyerName }}</p>
```

Comment:
```suggestion
      <p class="govuk-body" data-cy="deal-subheading"><a href="/case/{{ dealId }}/deal" data-cy="deal-subheading-link"><span class="govuk-visually-hidden">View deal</span> {{ deal.details.ukefDealId }} <span class="govuk-visually-hidden">details</span></a> {{ deal.submissionDetails.supplierName }} > {{ deal.submissionDetails.buyerName }}</p>
```

Reply:
I think this just makes it slightly clearer where the spaces are and more consistent with the others.